### ISSUE TITLE
Centos-XFCE won't start up

### ISSUE THREAD

**Issue by toschneck** [ConSol/docker-headless-vnc-container#4]
- Window-Manager is not up
- No control via VNC
- NoVNC won't startup

``` .bash
(xfsettingsd:71): xfsettingsd-WARNING **: Failed to get the _NET_NUMBER_OF_DESKTOPS property.

(xfwm4:62): xfwm4-WARNING **: Failed to connect to session manager: Failed to connect to the session manager: SESSION_MANAGER environment variable not defined
/etc/xdg/xfce4/xinitrc: line 172: which: command not found
** Message: First Orage start. Searching default timezone.
** Message: Default timezone set to Zulu.
** Message: Orage **: 15:53:54  wakeup timer init 0
Failed to connect to session manager: Failed to connect to the session manager: SESSION_MANAGER environment variable not defined
** Message: Orage **: 15:53:54  Created alarm list for main Orage file:
** Message: Orage **: 15:53:54          Added 0 alarms. Processed 0 events.
** Message: Orage **: 15:53:54          Found 0 alarms of which 0 are active. (Searched 0 recurring alarms.)

** (orage:93): WARNING **: Failed to connect to D-BUS daemon: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory

```

![issue](https://cloud.githubusercontent.com/assets/4659261/19642863/95115478-99e7-11e6-87ad-d8f6570cdd11.png)

**Comment by toschneck** [comment]
After fix, remove also fix from sakuli commit: https://github.com/ConSol/sakuli/commit/e758dbea607f9f9cd25676776c5afed443a95105 
```.bash
# TODO: remove when https://github.com/ConSol/docker-headless-vnc-container/issues/4 fixed and `1.0.2` released
ENV CHROMIUM_VERSION 53.0.2785.116-1
RUN yum erase -y google-chrome \
    && yum install -y http://install.linux.ncsu.edu/pub/yum/itecs/public/chromium/7/x86_64/chromium-$CHROMIUM_VERSION.el7.centos.x86_64.rpm \
    && yum clean all \
    && ln -s /usr/bin/chromium-browser /usr/bin/google-chrome
# fix to start chromium in a Docker container, see https://github.com/ConSol/docker-headless-vnc-container/issues/2
RUN sed -i -e 's/exec $CMD_PREFIX "$HERE\/chrome"/exec $CMD_PREFIX "$HERE\/chrome" $CHROMIUM_FLAGS/' /opt/chromium/chrome-wrapper
ENV CHROMIUM_FLAGS "--no-sandbox --start-maximized --user-data-dir"
```

**Comment by rbo** [comment]
@toschneck please check my branch rbo/dev, there are some changes and xfce4 works very well for me.
